lab-yunjoo/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+//imporing the greeting method from the greetings module
+const greet = require(__dirname + '/lib/greet').greet;
+//declear a const variable main
+//assigin module.exports and main the value of a function
+const main = module.exports = function(){
+  //return the value of greet with the argv[2] as an argument
+  return greet(process.argv[2]);
+};
+//console.log the result of
+console.log(main());
+
+// 'use strict';
```

Comment:
Its ok to comment out code while your working on a project, however before you make a pull request you will want to remove the commented out code. 

In the industry they will not allow commented out code into pull request, because that pull requset gets merged into the product.